### PR TITLE
new flow presubmit jobs for terraform-provider

### PR DESCRIPTION
**Pull request by amalykhi**

### CODE REVIEW

**Comment by amalykhi** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-ad-critical-high-f3

**Comment by amalykhi** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-pl-critical-high-f3

**Comment by amalykhi** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-ad-critical-high-presubmit

**Comment by amalykhi** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-pl-critical-high-presubmit

**Comment by amalykhi** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-ad-critical-high-presubmit

**Comment by amalykhi** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-v1.4.0-branch-rosa-sts-ad-critical-high-presubmit

**Comment by amalykhi** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-v1.4.0-branch-rosa-sts-ad-critical-high-presubmit

**Comment by amalykhi** [comment]
/pj-rehearse  pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-ad-critical-high-presubmit

**Comment by amalykhi** [comment]
/pj-rehearse  pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-pl-critical-high-presubmit

File: ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml [hunk 1]
```diff
@@ -148,6 +104,30 @@ tests:
     test:
     - ref: rhcs-e2e-tests
     workflow: rhcs-aws-sts
+- as: rosa-sts-ad-critical-high-presubmit
```
Comment (bardielle) [inline ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml:107]: Please add a description 
Also, what is it `ad`?

Comment (amalykhi) [reply ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml:106]: unfortunately, all the comments deleted by the script(

**Pull request merged**